force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +204,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String repeat;
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
+        for (Integer i = 1; i < repeatCount; i++) {
+            if(i == 1){
+                repeat = inputString;
+            }
+            repeat = repeat + '; ' + inputString;
             // If it is not the last iteration, add a semicolon to separate the strings
         }
-        */
-
         // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        return repeat; // Replace null with the variable you used to store the result
```

Comment:
Take a look at the Sring methods repeat and removeEnd.  

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_string.htm